sky/build/sky_precompilation_sdk.gni

```diff
@@ -34,6 +34,21 @@ template("sky_precompilation_sdk") {
     deps = [  snapshotter_target  ]
   }
 
+  script_snapshotter_copy_gen_target_name =
+      target_name + "_copy_script_snapshotter"
+  copy(script_snapshotter_copy_gen_target_name) {
+    script_snapshotter_target =
+        "//sky/tools/sky_snapshot($host_toolchain)"
+    script_snapshotter_directory =
+        get_label_info(script_snapshotter_target, "root_out_dir")
+    script_snapshotter_name = get_label_info(script_snapshotter_target, "name")
+
+    sources = [  "$script_snapshotter_directory/$script_snapshotter_name"  ]
+    outputs = [  "$sdk_dir/$tools_dir/ScriptSnapshotter"  ]
+
+    deps = [  script_snapshotter_target  ]
+  }
```

Comment:
Why not use the copy of sky_snapshot that's already in the `bin/cache` directory?